### DISCUSSION TITLE
使用docker进阶部署后，env中的apikey和proxyurl失效

### DISCUSSION THREAD

**Discussion by Zoumachuan** [lobehub/lobehub#3830] [General Question] [answered]
![image](https://github.com/user-attachments/assets/ea8fbbf0-0bef-4739-8eaa-3e1984435f02)
以上为完整env文件的信息，看issue里也有人发过类似的问题，但仍然没有解决

**Comment by SpeedupMaster** [comment, marked as answer]
OPEANAI_API_KEY 这个有问题，改成OPENAI_API_KEY

**Reply by Zoumachuan** [reply]
有点搞
